packages/database/src/core/util/util.ts

```diff
@@ -410,9 +406,11 @@ export const each = function(
      * a single impl that does a key, value callback. So we invert
      * to not have to touch the `each` code points
      */
-    forEach(obj, (key: any, val: any) => fn(val, key));
+    for (const [key, value] of Object.entries(obj)) {
```

Comment:
Please make sure that you do not create extra copies/loops in these utility functions.

Reply:
Done.